docs/vectorizer/document-embeddings.md

```diff
@@ -11,7 +11,7 @@ Then you can [query the generated embeddings](#query-document-embeddings) using
 
 In this guide, we'll also cover how to [monitor and troubleshoot](#monitoring-and-troubleshooting) your vectorizers.
 
-If you are storing documents in AWS S3, you can use the [S3 documentation](s3-documents.md) to learn more about how to configure S3 for document storage.
+If you are storing documents in AWS S3, you can use the [S3 documentation](s3-documents.md) to learn more about how to configure S3 for document storage and synchronize your S3 buckets with your document table.
```

Comment:
I would also put a reference to this in the `Setting up document storage` section